src/wallet/wallet_rpc_server.cpp

```diff
@@ -635,6 +636,18 @@ namespace tools
         tx_to_blob(ptx_vector.back().tx, blob);
         res.tx_blob = epee::string_tools::buff_to_hex_nodelimer(blob);
       }
+      if (req.get_tx_metadata)
+      {
+        std::ostringstream oss;
+        binary_archive<true> ar(oss);
+        ::serialization::serialize(ar, const_cast<tools::wallet2::pending_tx&>(ptx_vector.back()));
```

Comment:
This `const_cast` seems unnecessary.

Reply:
I made it const, as it should logically be.

Reply:
I see the change `for (const auto & ptx : ptx_vector)` in the other parts of this patch, but for this particular section `ptx_vector` is required to be non-`const`.
Not a big deal at all, but I just got curious.

Reply:
Oh, I see now. Done.